vendor/TraceKit/tracekit.js

```diff
@@ -1,7 +1,12 @@
 'use strict';
 
 /*
- TraceKit - Cross brower stack traces - github.com/occ/TraceKit
+ TraceKit - Cross brower stack traces
+
+ This was originally forked from github.com/occ/TraceKit, but has since been
+ largely re-written and is now maintained as part of raven-js.  Tests for
+ this are in test/vendor.
```

Comment:
:-)  Yeah, it took me a while to figure out where I needed to patch the code, so I figured I'd try and save future new contribs the same pain.